faq/views.py

```diff
@@ -15,38 +14,72 @@ def get_queryset(self):
         return Category.objects.prefetch_related('questions')
 
 
-class FAQAdminView(PermissionRequiredMixin, ListView):
-    model = Question
-    template_name = "faq/faq_admin.html"
-    context_object_name = "question_list"
+class FAQAdminPanelView(PermissionRequiredMixin, TemplateView):
+    template_name = 'faq/faq_admin_panel.html'
 
     def has_permission(self):
         return has_any_faq_permissions(self.request.user)
 
 
+class QuestionAdminView(PermissionRequiredMixin, ListView):
+    permission_required = ('faq.edit_question',)
```

Comment:
The permission created by default by Django is called `change_[model]`:
```suggestion
    permission_required = ('faq.change_question',)
```
One way of checking which permissions exist, is to open the `auth_permission` table in the database viewer in PyCharm 🙂 Another (preferred) way, is to write tests for this - which *should* always be done, but can be skipped for this PR, as it's more important that this feature goes live sooner rather than later 😅